boto/beanstalk/layer1.py

```diff
@@ -772,6 +772,8 @@ def rebuild_environment(self, environment_id=None, environment_name=None):
             Condition: You must specify either this or an EnvironmentId, or
             both.  If you do not specify either, AWS Elastic Beanstalk returns
             MissingRequiredParameter error.
+            If the environment hadn't be created or is terminated returns 
```

Comment:
Can you please clarify why this might happen? What could cause environment creation to fail?

Reply:
I just want to clarify that the environment must already exist and be in a not terminated state.
My use case is that I wanted to use it to rebuild in the morning environment I terminated in the night, that raise this InvalidParameterValue that wasn't documented.

Reply:
(FYI I haven't worked with this part of the code, which is why I'm asking these questions.)

The thing that confused me is that the function is called "rebuild_environment", so at first I thought your comment was saying the function wasn't able to rebuild the environment, rather than there was a problem with the environment you passed in.

How about this:
  - Move this to a raises: clause below
  - In that clause say "InvalidParameterValue if environment_name doesn't refer to a currently active environment."